cdisc_rules_engine/services/data_services/dummy_data_service.py

```diff
@@ -127,7 +127,7 @@ def get_define_xml_contents(self, dataset_name: str) -> bytes:
             with open(dataset_name, "rb") as f:
                 return f.read()
 
-        return bytes(self.define_xml)
+        return self.define_xml.encode()
```

Comment:
What is this change for?

Reply:
I was getting this error when running from the rule tester: `string argument without an encoding.`
str.encode() provides a default encoding

Reply:
Doesn't this change the return type of the method. I think this get define_xml method is expected to return bytes in the local data service. I think dummy data service should match

Reply:
no, they both do the same thing and return bytes.
`self.define_xml.encode()` is the same as `bytes(self.define_xml, 'utf-8')`